src/codeu/chat/client/commandline/Chat.java

```diff
@@ -181,6 +181,20 @@ private UserContext findUser(String name) {
       }
     });
 
+    panel.register(“info”, new Panel.Command() {
+      @Override
+      public void invoke(Scanner args) {
+        final ServerInfo info = context.getInfo();
+        if (info == null) {
+          // Communicate error to user - the server did not send us a valid
+            // info object.
+        } else {
+          // Print the server info to the user in a pretty way
```

Comment:
you'll need to print (current time - server start time) that you get from the ServerInfo object